### PR TITLE
Adopt the Open Code of Conduct

### PR DESCRIPTION
**Pull request by islemaster**
Adds the [Open Code of Conduct](http://todogroup.org/opencodeofconduct/) to our contributor guidelines.

### CODE REVIEW

**Comment by islemaster** [comment]
FYI @alicesteinglass @tanyaparker

**Comment by bcjordan** [comment]
LGTM!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # Contributing
 
-We'd love to have you join our group of contributors! Please e-mail your areas of interest and your availability to Alice (alice@code.org), and we’ll be happy to match you with a project. You can start setting up with these next steps.
+We'd love to have you join our group of contributors! Please e-mail your areas of interest and your availability to Alice (alice@code.org), and we’ll be happy to match you with a project.
```
Comment (ashercodeorg) [inline CONTRIBUTING.md]: Is it possible to use an alias (mailing group?) rather than a hard-coded user email?

Comment (tanyaparker) [inline CONTRIBUTING.md]: Good suggestion! I'll ask Lian.

**Pull request merged**